src/uid2Sdk.ts

```diff
@@ -396,20 +380,26 @@ export abstract class UID2SdkBase {
     this.setIdentity(cstgResult.identity);
   }
 
-  private throwIfInitNotComplete(message: string) {
+  protected throwIfInitNotComplete(message: string) {
     if (!this._initComplete) {
       throw new Error(message);
     }
   }
 }
 
 export class UID2 extends UID2SdkBase {
-  get product() {
-    return 'UID2';
-  }
+  // Deprecated. Integrators should never access the cookie directly!
   static get COOKIE_NAME() {
     return '__uid_2';
   }
+  private static get Uid2Details(): ProductDetails {
+    return {
+      name: 'UID2',
+      defaultBaseUrl: 'https://prod.uidapi.com',
+      localStorageKey: 'UID2-sdk-identity',
```

Comment:
This value is now defined in `mocks.ts` - is it worth using the same variable or is this intentional?

Reply:
It's intentional - I want to avoid exposing this value anywhere, and I think the test should be validating the value. I.e. if someone accidentally changes this value, the test should fail because it would be stored in the wrong place.